project/visualization/utils.py

```diff
@@ -1,16 +1,21 @@
 from __future__ import division
-from functools import reduce
-from io import BytesIO
+
 import operator
 import re
+from functools import reduce
+from io import BytesIO
 
+import django.db.models.fields as model_fields
+from PIL import Image as PILImage
+from PIL import ImageFile
 from django.conf import settings
 from django.core.files.storage import get_storage_class
 from django.db.models import Q
-import django.db.models.fields as model_fields
-from PIL import Image as PILImage
+
 from images.models import Point, Image, Metadata
 
+ImageFile.LOAD_TRUNCATED_IMAGES = True
```

Comment:
@StephenChan : I added this here, but perhaps better to move to project level config? (`config/settings/base.py`)?

Reply:
Moving to `config/settings/base.py` seems better, yeah. I just tried that and it works, so I'll add that change to this PR.